<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Suraj Yadav</author>
 <class>PolygonWindow</class>
 <widget class="QMainWindow" name="PolygonWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>568</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CGAL 2D Polygon</string>
  </property>
  <property name="windowIcon">
   <iconset resource="C:/ExtLibs/CGAL-4.9/demo/resources/CGAL.qrc">
    <normaloff>:/cgal/logos/cgal_icon</normaloff>:/cgal/logos/cgal_icon</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="focusPolicy">
        <enum>Qt::StrongFocus</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOn</enum>
       </property>
       <property name="transformationAnchor">
        <enum>QGraphicsView::NoAnchor</enum>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="fileToolBar">
   <property name="windowTitle">
    <string>File Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionClear"/>
   <addaction name="actionLoadPolygon"/>
   <addaction name="actionSavePolygon"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>Visualization Tools</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRecenter"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>568</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="actionLoadPolygon"/>
    <addaction name="actionSavePolygon"/>
    <addaction name="action_Export_SVG"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>&amp;Algorithms</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionRecenter"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionAboutCGAL">
   <property name="text">
    <string>About &amp;CGAL</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="C:/ExtLibs/CGAL-4.9/demo/icons/File.qrc">
     <normaloff>:/cgal/fileToolbar/fileNew.png</normaloff>:/cgal/fileToolbar/fileNew.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionLoadPolygon">
   <property name="icon">
    <iconset resource="C:/ExtLibs/CGAL-4.9/demo/icons/File.qrc">
     <normaloff>:/cgal/fileToolbar/fileOpen.png</normaloff>:/cgal/fileToolbar/fileOpen.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Load Polygon</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
  <action name="actionSavePolygon">
   <property name="icon">
    <iconset resource="C:/ExtLibs/CGAL-4.9/demo/icons/File.qrc">
     <normaloff>:/cgal/fileToolbar/fileSave.png</normaloff>:/cgal/fileToolbar/fileSave.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save Polygon</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionRecenter">
   <property name="icon">
    <iconset>
     <normaloff>:/cgal/Input/zoom-best-fit</normaloff>:/cgal/Input/zoom-best-fit</iconset>
   </property>
   <property name="text">
    <string>Re&amp;center the viewport</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionCreateInputPolygon">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Create Input Polygon</string>
   </property>
  </action>
  <action name="action_Export_SVG">
   <property name="text">
    <string>&amp;Export SVG...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Polygon.qrc"/>
  <include location="C:/ExtLibs/CGAL-4.9/demo/icons/File.qrc"/>
  <include location="C:/ExtLibs/CGAL-4.9/demo/resources/CGAL.qrc"/>
  <include location="C:/ExtLibs/CGAL-4.9/demo/icons/File.qrc"/>
 </resources>
 <connections/>
</ui>
